schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE `vet_clinic`.`animals` (
+  `id` INT NOT NULL,
+  `name` VARCHAR(45) NULL,
+  `date_of_birth` DATETIME NULL,
+  `escape_attemts` INT NULL,
+  `neutered` TINYINT NULL,
+  `weight_kg` DECIMAL(45) NULL,
+  PRIMARY KEY (`id`));
```

Comment:
* Kindly create your animal table following these steps and your table should be named animal as stated in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md#:~:text=Create%20a%20table%20named,weight_kg%3A%20decimal)


> Create a table named animals with the following columns and types:

> id: integer
> name: string
> date_of_birth: date
> escape_attempts: integer
> neutered: boolean
> weight_kg: decimal